vmm/src/vstate.rs

```diff
@@ -192,6 +197,8 @@ pub struct Vcpu {
     cpuid: CpuId,
     fd: VcpuFd,
     id: u8,
+    start_instance_request_cpu_ts: u64,
```

Comment:
I think these two look really out of place as members of `Vcpu`. If there's some compelling reason against leaving them as globals in `lib.rs` as before, we should pass them as parameters to `run()` or something like that.

Reply:
I agree that they look kinda out of place inside the `Vcpu`, but passing them as parameters involves loading them as parameters for quite a number of functions going down the stack to where they're actually used.

I'm working on some other code changes in this area and the only *nice* ways are to have them as:
- local object properties, or
- process-level global state.

I can argue that even though they *look out of place*, they are still properties of the `Vcpu` as they hold the TS for when *that `Vcpu` object instance* was created and are used to provide functionality for *that particular `Vcpu` instance*.

Reply:
Changed to creation timestamp as member of vcpu